examples/gno.land/r/sys/users/verifier.gno

```diff
@@ -0,0 +1,36 @@
+package users
+
+import (
+	"std"
+)
+
+type VerifyNameFunc func(enabled bool, address std.Address, name string) bool
```

Comment:
Unexport this as well. If we ever want to update the function, we will do so with a custom script managed manually.

Reply:
Done